src/acceptance_tests/uaa-release_test.go

```diff
@@ -140,9 +140,12 @@ var _ = Describe("UaaRelease", func() {
 
 	})
 
-	logLineWithoutTimestampRegex := ` uaa.* - \d+ \[([^]]+)\] .... (DEBUG|\sINFO|\sWARN) --- .+: .+`
+	logLineWithoutTimestampRegex := ` uaa.* - \d+ \[([^]]+)\] - \[[^]]+\] .... (DEBUG|\sINFO|\sWARN) --- .+: .+`
 
-	DescribeTable("UAA log format", func(uaaLogFormat string, optFiles ...string) {
+	DescribeTable("UAA log format", func(
+		uaaLogFormat string,
+		optFiles ...string,
+	) {
```

Comment:
I'd rather not see this kind of reformatting as part of the PR.

Reply:
Hmmm, interesting. For Go, the reformatting is usually automatic and something that's very difficult to suppress. So the code this was changed probably wasn't changed with an IDE.